### PR TITLE
Date and datetime widgets ported to Fluent UI

### PR DESCRIPTION
**Pull request by sramaswami11**
### Reasons for making this change

Ported date and datetime widgets

image

### Checklist

* [ ] **I'm updating documentation**
  - [ ] I've [checked the rendering](https://react-jsonschema-form.readthedocs.io/en/latest/#contributing) of the Markdown text I've added
* [ ] **I'm adding or updating code**
  - [ ] I've added and/or updated tests
  - [ ] I've updated [docs](https://react-jsonschema-form.readthedocs.io/) if needed
* [ ] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature

### CODE REVIEW

File: packages/fluent-ui/src/DateTimeWidget/DateTimeWidget.tsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ const DateTimeWidget = ({
       disabled={disabled}
       readOnly={readonly}
       name={name}
-      type={schema.type as string}
+      type="datetime-local"
```
Comment (epicfaace) [inline packages/fluent-ui/src/DateTimeWidget/DateTimeWidget.tsx]: Can you just render a `<TextWidget>` here?

Comment (epicfaace) [reply packages/fluent-ui/src/DateTimeWidget/DateTimeWidget.tsx]: See https://github.com/epicfaace/react-jsonschema-form/blob/395d1726f15fede459bb21ee23d4c55331285dd8/packages/fluent-ui/src/TextareaWidget/TextareaWidget.tsx

**Review by epicfaace — COMMENTED** [review]
Can you fix the widths so that datetimewidget is a bit less wide?

File: packages/fluent-ui/src/UpDownWidget/UpDownWidget.tsx [hunk 2]
```diff
@@ -91,26 +91,28 @@ WidgetProps) => {
...
       <SpinButton
+        id={id}
         min={min}
         max={max}
         step={step}
         incrementButtonAriaLabel={"Increase value by 1"}
         decrementButtonAriaLabel={"Decrease value by 1"}
         disabled={disabled || readonly}
-        value={value || (value === 0) ? value : ""}
+        value={value || value === 0 ? value : ""}
```
Comment (epicfaace) [inline packages/fluent-ui/src/UpDownWidget/UpDownWidget.tsx:109]: Can you revert this line?

File: packages/fluent-ui/src/DateWidget/DateWidget.tsx [hunk 3]
```diff
@@ -63,7 +63,7 @@ const DayPickerStrings: IDatePickerStrings = {
 const controlClass = mergeStyleSets({
   control: {
     margin: "0 0 15px 0",
-    maxWidth: "300px",
+    maxWidth: "700px",
```
Comment (epicfaace) [inline packages/fluent-ui/src/DateWidget/DateWidget.tsx]: Why hardcode this width?

Comment (sramaswami11) [reply packages/fluent-ui/src/DateWidget/DateWidget.tsx]: To make the width match with DateTime field , in the original code the width was set to 300, I bumped it up to 700

File: packages/fluent-ui/src/DateTimeWidget/DateTimeWidget.tsx (hunk 1 above)
Comment (sramaswami11) [reply packages/fluent-ui/src/DateTimeWidget/DateTimeWidget.tsx]: I did.

File: packages/fluent-ui/src/DateWidget/DateWidget.tsx (hunk 3 above)
Comment (sramaswami11) [reply packages/fluent-ui/src/DateWidget/DateWidget.tsx]: Removed the hard coded value

**Pull request merged**